v1.0 web (#5)
Picovoice/koala: README.md

    - [Demos](#demos)
        - [Python](#python-demos)
        - [C](#c-demos)
        - [Web](#web-demo)
    - [SDKs](#sdks)
        - [Python](#python)

>         - [Web](#web)

Reviewer: looks like you removed the link to the C section?

pv_koala_delete(handle);
```

### Web

Install the web SDK using yarn:

```console
yarn add @picovoice/koala-web
```

or using npm:

```console
npm install --save @picovoice/koala-web
```

Create an instance of the engine using `KoalaWorker` and enhance audio in real-time:

```typescript
import { Koala } from "@picovoice/koala-web";
import koalaParams from "${PATH_TO_BASE64_KOALA_PARAMS}";

function processCallback(enhancedPcm) {
  // do something with enhancedPcm
}

> function getAudioData(): Int16Array {
> ... // function to get audio data
>   return new Int16Array();
> }

Reviewer: be more clear to call it "AudioFrame"
```suggestion
function getAudioFrame(): Int16Array {
  // ... function to get a frame of audio
}
```

const koala = await KoalaWorker.create(
  "${ACCESS_KEY}",
  processCallback,
  { base64: koalaParams },
);

await koala.reset();
for (;;) {

>     await koala.process(getAudioData());

Reviewer: ```suggestion
    await koala.process(getAudioFrame());
```
